### PR TITLE
WT-11832 Add the strict option to the wt tool verify command

### PR DESCRIPTION
**Pull request by etienneptl**
The `strict` command can be enabled with the `-S` option when using the `verify` command:
```
./wt -h <db_path> -r verify -S
```

### CODE REVIEW

File: src/utilities/util_verify.c [hunk 1]
```diff
@@ -94,6 +94,9 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
             else
                 return (usage());
             break;
+        case 'e':
```
Comment (kommiharibabu) [inline src/utilities/util_verify.c:97]: instead of 'e' how about using 'S'?

Comment (etienneptl) [reply src/utilities/util_verify.c:97]: Sure that would work too, I was thinking as `e` for `error`.

Comment (etienneptl) [reply src/utilities/util_verify.c]: See 762ad085118c4e6ad5bf6d21558a16c2e5cf373c

File: src/utilities/util_verify.c [hunk 2]
```diff
@@ -62,12 +62,12 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
...
     bool abort_on_error, do_not_clear_txn_id, dump_address, dump_app_data, dump_blocks, dump_layout,
-      dump_pages, read_corrupt, stable_timestamp;
+      dump_pages, read_corrupt, stable_timestamp, strict;
 
     abort_on_error = do_not_clear_txn_id = dump_address = dump_app_data = dump_blocks =
-      dump_layout = dump_pages = read_corrupt = stable_timestamp = false;
+      dump_layout = dump_pages = read_corrupt = stable_timestamp = strict = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "acd:stu?")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "acd:eStu?")) != EOF)
```
Comment (mm-ng) [inline src/utilities/util_verify.c:70]: I think this `e` needs to be removed here.

Comment (etienneptl) [reply src/utilities/util_verify.c]: Thank you, removed in 9fb9e84d3086327ce1e418ea4f411b6550602b99

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Review by mm-ng — APPROVED** [review]
LGTM :)

**Review by Sean04 — APPROVED** [review]
LGTM

**Comment by etienneptl** [comment]
evergreen merge

**Pull request merged**